pkg/security/admission/admission_test.go

```diff
@@ -1211,6 +1256,38 @@ func createSCCListerAndIndexer(t *testing.T, sccs []*securityapi.SecurityContext
 }
 
 func createSCCLister(t *testing.T, sccs []*securityapi.SecurityContextConstraints) securitylisters.SecurityContextConstraintsLister {
+	t.Helper()
 	lister, _ := createSCCListerAndIndexer(t, sccs)
 	return lister
 }
+
+type sccTestAuthorizer struct {
+	t *testing.T
+
+	// this user, in this namespace, can use this SCC
+	user      string
+	namespace string
+	scc       string
+}
+
+func (s *sccTestAuthorizer) Authorize(a authorizer.Attributes) (authorizer.Decision, string, error) {
+	s.t.Helper()
```

Comment:
out of curiosity, does this really help?  Seems like you wouldn't want to skip writing this in line output.